core/src/tf_lib.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * @license
+ * Copyright 2018 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import * as tf from '@tensorflow/tfjs';
+
+export function split(
```

Comment:
is this something we should contribute back to tfjs?

Reply:
Working on it now :)

Reply:
https://github.com/tensorflow/tfjs-core/pull/930